index.html

```diff
@@ -301,7 +301,7 @@ <h3>
               />
             </form>
           </div>
-          <img
+          <!-- <img
```

Comment:
- [ ] (OPTIONAL) Kindly remove all pieces of commented code 👍 to keep your work neat